components/Renderers/Writing.tsx

```
@@ -22,20 +24,33 @@ const Writing = ({ fileUrl, description }: WritingProps) => {
+        shouldCenter && "flex items-center justify-center",
+        isShortText && "text-xl md:text-3xl",
+      )}
+      ref={containerRef as any}
+    >
       <div
-        className="font-spectral bg-grey-eggshell p-2 text-md !normal-case text-left"
+        className={cn(
+          "bg-grey-eggshell p-2 !normal-case text-left",
+          !Boolean(containerHeight) && "opacity-0",
+        )}
         dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)